.gitignore

```diff
@@ -0,0 +1,24 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Distribution / packaging
+build/
+dist/
+pip-wheel-metadata/
+.eggs/
+*.egg-info/
+*.egg
+
+# Unit test / coverage reports
+.tox
+.cache
+.coverage
+
+# setuptools_scm maintains version.py
+src/discreetly/version.py
```

Comment:
I actually don't have this, maybe I install the lib incorrectly ?

Reply:
This should be `src/c360_client/version.py`

I believe the `version.py` is created on release and is therefore gitignored.

See https://github.com/pypa/setuptools_scm#configuration-parameters

Reply:
addressed in https://github.com/tra-sg/c360-python-client/pull/1/commits/8e0c6e75592f031e6787175ffce867b2dc9fd57b